tests/unit/s2n_aead_aes_test.c

```diff
@@ -239,7 +240,7 @@ int main(int argc, char **argv)
             EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->header_in));
             EXPECT_SUCCESS(s2n_stuffer_copy(&conn->out, &conn->header_in, 5));
```

Comment:
Is there a macro that can replace the constant "5"? Here and on line 417

Reply:
This is the TLS header length. It is turns out there is: `S2N_TLS_RECORD_HEADER_LENGTH`.
There are quite a few of these instances in both unit tests files. I think I found all of them.